README.md

```diff
@@ -59,6 +61,31 @@ Install specific files in Dom0:
 sudo qubesctl state.apply dotfiles.copy-dom0,dotfiles.copy-sh,dotfiles.copy-vim,dotfiles.copy-x11
 ```
 
+#### Pillar
+
+With Salt, the execution of each state can be opt out via the dotfiles pillar.
+You will need a top and a sls file in your pillar_roots (default to /srv/pillar,
+/srv/pillar/base, can be configured in /etc/salt/minion.d).
```

Comment:
Mention paths inside inline code notation such as `/srv/pillar`. [Qubes Builder readme](https://github.com/ben-grande/qusal/tree/main/salt/qubes-builder) example where I use that syntax to mention files

Reply:
Sorry, after looking at the qubes builder readme, I still don't understand what you are asking here.

Reply:
With Salt, the execution of each state can be opt out via the dotfiles pillar.
You will need a top and a sls file in your pillar_roots (default to `/srv/pillar`,
`/srv/pillar/base`, which can be configured in `/etc/salt/minion.d`).

---

I guess we should mention only `/srv/pillar/qusal` and enable it in Qusal's minon.d file do there is no need to mention `/etc/salt/minion.d`.

Reply:
Ok I will inline paths.

>  guess we should mention only /srv/pillar/qusal and enable it in Qusal's minon.d file do there is no need to mention /etc/salt/minion.d.

Ok but that imply to specify that the user need to execute the setup script to make it works:

You will need a top file and a sls file in your pillar_roots, when following [the qusal's installation instructions](https://github.com/ben-grande/qusal/blob/main/docs/INSTALL.md), `/srv/pillar/qusal` will be added to you pillar_roots.